gradle/libs.versions.toml

```diff
@@ -16,7 +16,7 @@ bcpkix = "1.70"
 micronaut-test = "4.0.0-M3"
 micronaut-multitenancy = "5.0.0-M2"
 micronaut-reactor = "3.0.0-M1"
-micronaut-serde = "2.0.0-M1"
+micronaut-serde = "2.0.0-M4"
```

Comment:
Serde M4 includes https://github.com/micronaut-projects/micronaut-serialization/pull/439 Thanks for the quick fix @yawkat